client/src/LayoutComponents/AboutUs/AboutUs.css

```diff
@@ -0,0 +1,50 @@
+.container{
```

Comment:
@kholoudfann These class names are a little generic and might also be used in other components (which would mean that these styles would accidentally apply to those components too), so it might be a good idea to make them more specific by adding an ".about-us" classname to the top-level div inside the component and then use it for all of these selectors:
`.about-us .container {`
`.about-us .title {`
and so on.

Reply:
thanks a lot, i will do that